server/src/controller/reviews.ts

```diff
@@ -1,16 +1,43 @@
 import { Request, Response } from 'express'
 import Review from '../model/reviews'
+import Meeting from '../model/meetings'
+import Member from '../model/members'
 import MeetingMemberMapping from '../model/meetingMemberMapping'
+import { Op } from 'sequelize'
 
 export async function getReviewByUser(req: Request, res: Response) {
```

Comment:
이것도 리뷰가 여러개가 리턴되니까 복수로 바꿔주면 좋을 것 같아요.
```suggestion
export async function getReviewsByUser(req: Request, res: Response) {
```